kubeflow/trainer/constants/constants.py

```diff
@@ -148,8 +148,8 @@
     *DEFAULT_COMMAND,
 )
 
-# The default name for the Torch runtime.
-TORCH_RUNTIME = "torch-distributed"
+# The default name for the ClusterTrainingRuntime.
+DEFAULT_TRAINING_RUNTIME = os.getenv("DEFAULT_TRAINING_RUNTIME", "torch-distributed")
```

Comment:
I also enable to override the default runtime via env variables, I believe it might be very useful.